spec/support/table_list.txt

```diff
@@ -0,0 +1,335 @@
+accounts
```

Comment:
Not sure I love authors having to keep this list up to date, but I can't think of a better way to do this.  @agrare Thoughts?